Source/WebKit/Shared/unix/BreakpadExceptionHandler.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 RDK Management
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#if ENABLE(BREAKPAD)
+#include <WebKit/WKBase.h>
+
+namespace WebKit {
+
+WK_EXPORT void installBreakpadExceptionHandler();
```

Comment:
Uh-oh, this is aproblem. This is an internal API: it is not exposed in the WPE public API. The `WK_EXPORT` here makes it accessible to tools and tests when running from the build directory, not to applications; when installed, the symbol will stripped away by our linker version script (webkitglib-symbols.map).

So this needs a rethink. Does the application really need to call this API itself? Probably better for WebKit to handle this itself?

Reply:
Well, sorry, I see you do use it in AuxiliaryProcessMain. So just remove the `WK_EXPORT` and we're good:

```suggestion
void installBreakpadExceptionHandler();
```